### PR TITLE
add: `cmake-deb`

### PR DESCRIPTION
**Pull request by Zahrun**
Did you see this coming?

### CODE REVIEW

File: packages/cmake-deb/cmake-deb.pacscript [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name="cmake-deb"
+gives="cmake"
+version="3.25.1"
+description="CMake data files (modules, templates and documentation)"
+pacdeps=("cmake-data-deb")
+codename="$(lsb_release -cs)"
+case ${codename} in
```
Comment (Henryws) [inline packages/cmake-deb/cmake-deb.pacscript:7]:
```suggestion
case "${codename}" in
```

**Review by Henryws — CHANGES_REQUESTED** [review]

**Review by Henryws — APPROVED** [review]

**Comment by Zahrun** [comment]
@Henryws 
```
CMake Error at CMakeLists.txt:1 (cmake_minimum_required):
  CMake 3.25.2 or higher is required.  You are running version 3.25.1
```
😂
I guess I will wait for the patch update then 😅

**Comment by Henryws** [comment]
Ok, make a new PR, and add `incompatible` to it.

**Comment by Zahrun** [comment]
No, I mean I wanted to use this updated `cmake` to update `obs-backgroundremoval-git`, but it requires an ever so slightly newer cmake 😅
It's ok, I guess debian and ubuntu will catch up in some time. then I'll update these pacscripts to 3.25.2

**Pull request merged**